src/status_im/contexts/profile/settings/header/style.cljs

```diff
@@ -23,13 +22,8 @@
    :flex-direction  :row
    :justify-content :space-between})
 
-(defn avatar-container
-  [theme scale-animation top-margin-animation side-margin-animation]
-  [{:transform     [{:scale scale-animation}]
-    :margin-top    top-margin-animation
-    :margin-left   side-margin-animation
-    :margin-bottom side-margin-animation}
-   {:align-items   :flex-start
-    :border-width  4
-    :border-color  (colors/theme-colors colors/border-avatar-light colors/neutral-80-opa-80 theme)
-    :border-radius 100}])
+(defn avatar-border-color
+  [theme]
+  (if platform/android?
+    colors/neutral-80-opa-80 ;; Fix is not needed because Android doesn't use blur
+    (colors/theme-colors colors/border-avatar-light colors/neutral-80-opa-80 theme)))
```

Comment:
Avatar's ring fix

Reply:
This ring I'm referring to is the one showed in PR's video:

https://github.com/user-attachments/assets/d366b167-c575-4f56-9fea-ef6254eb6c15

different to the identity ring